RecoParticleFlow/PFTracking/plugins/PFElecTkProducer.cc

```diff
@@ -186,7 +187,8 @@ PFElecTkProducer::produce(Event& iEvent, const EventSetup& iSetup)
       // remove fifth step tracks
       if( useFifthStepForEcalDriven_ == false
 	  || useFifthStepForTrackDriven_ == false) {
-	bool isFifthStepTrack = isFifthStep(kf_ref);
+	
+	bool isFifthStepTrack = PFTrackAlgoTools::isFifthStep(kf_ref->trackRef()->algo());
```

Comment:
It seems the definitiopn of fithStep [which is crazy naming ] has different meaning in different modules